bokeh/client/websocket.py

```diff
@@ -68,26 +60,11 @@ async def write_message(self, message: str | bytes, binary: bool = False, locked
         Bokeh Document lock.
 
         '''
-        def write_message_unlocked() -> Future[None]:
-            if self._socket.protocol is None:
-                # Tornado is maybe supposed to do this, but in fact it
-                # tries to do _socket.protocol.write_message when protocol
-                # is None and throws AttributeError or something. So avoid
-                # trying to write to the closed socket. There doesn't seem
-                # to be an obvious public function to check if the socket
-                # is closed.
-                raise WebSocketError("Connection to the server has been closed")
-
-            future = self._socket.write_message(message, binary)
-
-            # don't await this future or we're blocking on ourselves!
-            return future
-
```

Comment:
AFAICT all this was for ancient versions of Tornado. No tests are affected by removal, manual testing does not reveal any changes I can see, and explicit attempts to probe disconnection scenarios also don't reveal anything out of the ordinary or unwanted.